A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_DirectToPage.js

```diff
@@ -1,12 +1,16 @@
 class CDUDirectToPage {
     static ShowPage(mcdu, directWaypoint, wptsListIndex = 0) {
         mcdu.clearDisplay();
-        let directWaypointCell = " ";
+        let directWaypointCell = "[][color]blue";
+        let pageTitle = "DIR TO";
+        let dirETA = "----";
+        let dirDist = "---";
+        let directToLabel = "DIRECT TO[color]blue";
         if (directWaypoint) {
-            directWaypointCell = directWaypoint.ident;
+            directWaypointCell = directWaypoint.ident + "[color]yellow";
         }
         else if (mcdu.flightPlanManager.getDirectToTarget()) {
-            directWaypointCell = mcdu.flightPlanManager.getDirectToTarget().ident;
+            directWaypointCell = "[\xa0\xa0\xa0\xa0\xa0][color]blue";
```

Comment:
Is ` \xa0` actually working? I know `&nbsp;` doesn't.

Reply:
Yep, working fine in the sim - makes the spacing between the square brackets closer to the reference material.

Reply:
Example:
![image](https://user-images.githubusercontent.com/37539998/95001550-4e97c000-05cb-11eb-8dde-7f9766772afb.png)

Reply:
Sweet, useful to know for a lot of areas!